packages/atomic/src/components.d.ts

```diff
@@ -503,7 +503,7 @@ export namespace Components {
          */
```

Comment:
Not sure about these changes 😅

Reply:
That's weird I didn't touch those files, don't know how that happened...